### PR TITLE
Add ASN.1 pretty print for examples

### PR DESCRIPTION
**Pull request by seanturner**
Adding ASN.1 pretty print for examples.

### CODE REVIEW

File: draft-ietf-lamps-kyber-certificates.md [hunk 1]
```diff
@@ -338,41 +338,341 @@ The following is an example of a ML-KEM-512 private key with hex seed `0001…3f
...
+  NOTE: The private key is the seed and all three examples keys
+  use the same seed; therefore, the private above are the same
+  except for the OID used to represent the ML-KEM algorithm's
+  security strength.
+</aside>
+
 ## Example Public Key {#example-public}
 
-The following is the ML-KEM-512 public key corresponding to the private
+The following is the ML-KEM-512 public key corresponding to the public
```
Comment (seanturner) [inline draft-ietf-lamps-kyber-certificates.md:404]:
```suggestion
The following is the ML-KEM-512 private key corresponding to the public
```

File: draft-ietf-lamps-kyber-certificates.md [hunk 2]
```diff
@@ -338,41 +338,341 @@ The following is an example of a ML-KEM-512 private key with hex seed `0001…3f
...
+  NOTE: The private key is the seed and all three examples keys
+  use the same seed; therefore, the private above are the same
+  except for the OID used to represent the ML-KEM algorithm's
+  security strength.
+</aside>
+
 ## Example Public Key {#example-public}
 
-The following is the ML-KEM-512 public key corresponding to the private
+The following is the ML-KEM-512 private key corresponding to the public
```
Comment (seanturner) [inline draft-ietf-lamps-kyber-certificates.md:404]:
```suggestion
The following is the ML-KEM-512 public key corresponding to the private
```

File: draft-ietf-lamps-kyber-certificates.md [hunk 3]
```diff
@@ -359,20 +408,271 @@ key in the previous section.
...
+      :     38 a6 ba 98 7e c3 72 25 c7 b4 4c db 12 dd e4 53
+      :     9b 4a b0 82 36 36 83 f0 4b f7 a0 9c c5 c4 1d fe
+      :     83 0a 1b 16 2e 0b 32 43 34 36 2f 08 4a 14 46 77
+      :     23 34 4b ad d0 00 f8 d8 c5 37 c4 8f 99 8f 05 30
+      :     7c eb d1 ed e0 b8 1c 3b c5 9a 06 5a 1b 6d 63 b2
+      :     6c
+      :   }
+~~~
+
+The following is the ML-KEM-768 public key corresponding to the public
```
Comment (seanturner) [inline draft-ietf-lamps-kyber-certificates.md:470]:
```suggestion
The following is the ML-KEM-768 public key corresponding to the private
```

File: draft-ietf-lamps-kyber-certificates.md [hunk 4]
```diff
@@ -359,20 +408,271 @@ key in the previous section.
...
+      :     0d 77 47 cd d0 f5 b9 9a a8 c8 26 98 75 17 79 3a
+      :     aa 80 80 a0 b1 24 a8 55 8d f7 2b be 37 b7 5f 4e
+      :     db b6 be 82 16 d6 c6 33 fb 2b 22 80 e2 51 13 d8
+      :     69 5e 43 48 1c 3e eb 39 7e b1 92 50 52 29 b6 7a
+      :     20 1e a8 93 c3 e2 cb 32 da 8b c3 42 fa 4d ea 05
+      :     78
+      :   }
+~~~
+
+The following is the ML-KEM-1024 public key corresponding to the public
```
Comment (seanturner) [inline draft-ietf-lamps-kyber-certificates.md:561]:
```suggestion
The following is the ML-KEM-1024 public key corresponding to the private
```

**Comment by bwesterb** [comment]
How did you create these? Would be cool to auto format with a simple `make` from the generated examples in case we still need any changes.

**Review by bwesterb — APPROVED** [review]

**Comment by dconnolly** [comment]
> How did you create these? Would be cool to auto format with a simple `make` from the generated examples in case we still need any changes.

yah these are cool

**Comment by seanturner** [comment]
I used openssl asn1parse and then had to hand jam 'em. I got the idea from a @danvangeest was working on. Maybe he's got some other pretty print utility?

**Comment by danvangeest** [comment]
For everyone else: `dumpasn1` Linux tool.  I had to mess with the command line parameters to make it nice for RFCs, possibly `-a -w66` or some other unintuitive number.

**Pull request merged**